mgc/terraform-provider-mgc/mgc/resources/mgc_dbaas_instances_backups.go

```diff
@@ -0,0 +1,219 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	mgcSdk "magalu.cloud/lib"
+	dbaasBackups "magalu.cloud/lib/products/dbaas/instances/backups"
+	"magalu.cloud/terraform-provider-mgc/mgc/client"
+	"magalu.cloud/terraform-provider-mgc/mgc/tfutil"
+)
+
+const backupStatusTimeout = 30 * time.Minute
```

Comment:
aumenta esse tempo... só pra não quebrarmos do nosso lado se houver algum problema lado server...
e ai, o usuario poderá utilizar o timeout da forma que achar melhor...
https://developer.hashicorp.com/terraform/plugin/sdkv2/resources/retries-and-customizable-timeouts